src/main/kotlin/gropius/model/issue/Label.kt

```diff
@@ -0,0 +1,42 @@
+package gropius.model.issue
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.generator.annotations.GraphQLType
+import gropius.model.architecture.Trackable
+import gropius.model.common.NamedSyncNode
+import io.github.graphglue.model.*
+import org.springframework.data.annotation.Transient
+import java.time.OffsetDateTime
+
+@DomainNode
+@GraphQLDescription(
+    """Label used to mark Issues with.
+    A Label consists of a name, a description and a color.
+    Issues are synced to all IMSProjects of Trackables they are part of.
```

Comment:
Do not state this here so sure...
```suggestion
    Issues may be synced to all IMSProjects of Trackables they are part of.
```